spk/nzbget/src/service-setup.sh

```diff
@@ -0,0 +1,84 @@
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
+NZBGET="${SYNOPKG_PKGDEST}/bin/nzbget"
+CFG_FILE="${SYNOPKG_PKGDEST}/var/nzbget.conf"
+TEMPLATE_CFG_FILE="${SYNOPKG_PKGDEST}/share/nzbget/nzbget.conf"
+UPGRADE_CFG_FILE="${TMP_DIR}/nzbget.conf"
+WEBDIR="${SYNOPKG_PKGDEST}/bin/webui"
+
+GROUP="sc-download"
+
+# Force-overwrite the PID-file and WebDir setting
+# These could change depending on previous package settings
+SERVICE_COMMAND="${NZBGET} -c ${CFG_FILE} -o WebDir=${WEBDIR} -o LockFile=${PID_FILE} -D"
+
+# Needed to force correct permissions, during update
+# Extract the right paths from config file
+if [ -r "${UPGRADE_CFG_FILE}" ]; then
+    DOWNLOAD_FOLDER=`grep -Po '(?<=MainDir=).*' ${UPGRADE_CFG_FILE}`
+    if [ -n "$(dirname "${DOWNLOAD_FOLDER}")" ]; then
+        SHARE_PATH=$(dirname "${DOWNLOAD_FOLDER}")
```

Comment:
This trick has a side effect that may be an issue later. `SHARE_PATH` is shorten to `volume1` (for default value) and is stored that way in `etc/installer-variable`.
Is there really no `service_xxx` hook function that may handle a specific call to `set_syno_permissions` ?